src/Kernel/CompiledCode.class.st

```diff
@@ -144,7 +144,7 @@ CompiledCode >> author [
 ]
 
 { #category : #accessing }
-CompiledCode >> bytecode [
+CompiledCode >> bytecodes [
```

Comment:
add bytecode to be backward compatible?

CompiledCode>> bytecode
    "deprecated"
    ^ self bytecodes